### ISSUE TITLE
Is there a reason /LASTLOG doesn't work?

### ISSUE THREAD

**Issue by smathy** [MacIrssi/MacIrssi#39]
Subject says it all.  I'd love to help make this work if it's something within my capabilities (I'm no big C wizz).

**Comment by ghost** [comment]
Yes, sadly. It's part of the TextUI subsystem, which MacIrssi doesn't use because it replaces it with the Cocoa interface.

**Comment by smathy** [comment]
So, it'd basically have to be implemented entirely from scratch?

**Comment by ghost** [comment]
Yeah, pretty much. Something I don't really have time to do myself at the moment. I know it's not the same thing but the beta versions have a search bar for searching the scrollback buffer.

**Comment by smathy** [comment]
Ok, yes, that's a great addition.  Thanks, I didn't know about that.

**Comment by ghost** [comment]
I don't know if it would make sense to wire /LASTLOG to show the search bar ... ?
